dotnet3.5/dynamodb/metadata.yaml

```diff
@@ -0,0 +1,262 @@
+files:
```

Comment:
Cleanup report found a couple missing files:
File 'https://github.com/awsdocs/aws-doc-sdk-examples/tree/master/dotnet3.5/Getting_Started_VS/Getting-Started-VS/Program.cs' reported in metadata does not exist in the repo.
File 'https://github.com/awsdocs/aws-doc-sdk-examples/tree/master/dotnet3.5/Getting_Started_VS/Getting_Started_VS_Tests/ProgramTests.cs' reported in metadata does not exist in the repo.

Reply:
Fixed